qt/widgets/common/inc/MantidQtWidgets/Common/MWRunFiles.ui

```diff
@@ -149,13 +149,16 @@
       <string/>
      </property>
      <property name="icon">
-      <iconset resource="../../../../images/images.qrc">
```

Comment:
Can I ask the reason that the resource file is removed? Do we not need it for the `LoadFile.png` reference?

Reply:
This appears to be a mistake from the changes made in QtDesigner. I'll revert this like back.

Reply:
Thanks!